packages/legend-application-studio/src/stores/editor/utils/ModelClassifierUtils.ts

```diff
@@ -69,6 +69,7 @@ export enum PACKAGEABLE_ELEMENT_TYPE {
   GENERATION_SPECIFICATION = 'GENERATION_SPECIFICATION',
   SECTION_INDEX = 'SECTION_INDEX',
   DATA = 'DATA',
+  SNOWFLAKE_APP = 'snowflakeApp',
```

Comment:
should we keep it in sync with the other formats?

Reply:
you are right